dynamiqs/plots/misc.py

```diff
@@ -0,0 +1,116 @@
+from __future__ import annotations
+
+import numpy as np
+from matplotlib.axes import Axes
+from matplotlib.colors import ListedColormap, LogNorm, Normalize
+
+from ..utils.tensor_types import ArrayLike, to_numpy
+from .utils import add_colorbar, fock_ticks, optax, sample_cmap
+
+__all__ = [
+    'plot_pwc_pulse',
+    'plot_fock',
+    'plot_focks',
+]
+
+
+@optax
+def plot_pwc_pulse(
```

Comment:
Maybe add a `set_xlim` on this plot as well so there are no blank spaces on right/left sides.

Reply:
It looks a little weird to me, maybe it's just because I'm not used to it.
![9f07ea9d-4a5f-4b56-a699-080ed70443fe](https://github.com/dynamiqs/dynamiqs/assets/38159029/7de468cd-df66-419e-a4cb-0bfa1c663e51)

If you prefer it like that I'll change.

Reply:
I changed to remove blank spaces in latest commit.

Reply:
I think I prefer it like that yes, but it's a matter of preference.